services/order.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime | None = None
+) -> Order:
+    user = get_user_model().objects.get(username=username)
+    with transaction.atomic():
+        order = Order.objects.create(user=user)
+
+        if date:
+            order.created_at = date
+        order.save()
+
+        for ticket in tickets:
+            row = ticket.get("row")
+            seat = ticket.get("seat")
+            movie_session = ticket.get("movie_session")
```

Comment:
Since this data is necessary to create a ticket, you can access a value directly using a key, without using get method